src/BenchmarksApps/Samples/Template/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+
+namespace Template
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateWebHostBuilder(args).Build().Run();
+        }
+
+        public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
+            WebHost.CreateDefaultBuilder(args)
+                .UseStartup<Startup>();
```

Comment:
Are you ok if I disable logging here also, such that it becomes opt-in (and documented) as logging adds some significant allocations (AsyncLocal to create scopes).

Reply:
The idea is that there is the least noise to the actual benchmarks.

Reply:
another good idea, I've disabled the logging. The RPS went from 200k to 230k ;)